### DISCUSSION TITLE
There is only a few PS_points inversed.

### DISCUSSION THREAD

**Discussion by gaiyf** [insarlab/MintPy#845] [Q&A]
i used 12 images to do this project , the ps_points is only a few. 
i want to know should i change some parameters or add more  images?
![geo_velocity](https://user-images.githubusercontent.com/57431511/189485999-07ced8b9-72c5-4a47-9da2-369e89c76c8d.png)

**Comment by yunjunz** [comment]
This is a big question. You may want to get familiar with how mintpy works, following the tutorial here:

https://github.com/insarlab/MintPy-tutorial#1-small-baseline-time-series-analysis-with-smallbaselineapppy
